### PR TITLE
21193 - Show Completed Cont In (T number)

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#21193
- Ticket #21193
- Added filing history component for Continuation In (Temp mode)
- Added unit tests

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
SB says, try this (with the temp URL):

https://dev.business.bcregistry.gov.bc.ca/TROBxAE4MA?filing_id=149467

File: src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        :filing="filing"
+        :index="index"
+      />
+
+      <FutureEffectivePaid
+        v-else-if="isFutureEffective"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:20]: See if you can test both of the above cases ^^

**Comment by ArwenQin** [comment]
Pending:


Completed:
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-669-eon0kpmn.web.app

File: src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+        <div class="reload-business-container text-center mt-6">
+          <v-btn
+            color="primary"
+            @click.stop="reloadWithBusinessId()"
+          >
+            <span>Retrieve Business Information</span>
+          </v-btn>
+        </div>
+      </div>
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:56]: Please show a screenshot of the above.

Also, please show a screenshot of the completed filing. I'm especially interesting if the court order info is displayed.

Comment (ArwenQin) [reply src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:56]: image

File: src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue (hunk 1 above)
Comment (ArwenQin) [reply src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:20]: > See if you can test both of the above cases ^^ and their corresponding bodies below.

I will add the tests tmr before I merge this PR. Thank you!

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:20]: No, I mean manually verify the screen. You'll have to create a future-effective continuation in to see this.

Comment (ArwenQin) [reply src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:20]: > No, I mean manually verify the screen. You'll have to create a future-effective continuation in to see this.



> No, I mean manually verify the screen. You'll have to create a future-effective continuation in to see this.

Got it, thank you. I'm able to test the 2 cases, it shows properly:
future-effective:


future-effective pending:

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/filings/ContinuationInApplication.vue:20]: Looks great. Resolving.

Go ahead and merge.

**Review by ketaki-deodhar — APPROVED** [review]
Got to this a bit late. Tested a bit and looks good!

**Pull request merged**